File: C/PRAK502-2210817310001-MuhammadRyanRizkyRahmadi.c
#include <stdio.h>
#include <math.h>
int rumus(int r, int y){
    return r - y;
}

int mutlak(int angka) {
    return fabs(angka);
}

int main() {
    int a, b, c, d;
        scanf("%d",&a);
        scanf("%d",&c);
        scanf("%d",&b);
        scanf("%d",&d);
        int Hasil = rumus(a,b) + rumus(c,d); 
        printf("%d",mutlak(Hasil));
    
    
    return 0;
}

File: Python/PRAK502-2210817310001-MuhammadRyanRizkyRahmadi.py
def hitung(r, y):
      return int(r) - int(y)

def mutlak(angka):
     return abs(angka)

for i in range(3):
    x1, y1, x2, y2 = input().split()
    hasil = hitung(x1, x2) + hitung(y1, y2)
    
    print(f'{mutlak(hasil)}\n')